VideoApp/VideoApp/SwiftUI/Screens/Room/RoomViewModel.swift

```diff
@@ -30,59 +30,52 @@ import Combine
 
     @Published var state = State.disconnected
     @Published var layout: Layout = .gallery
+    @Published var isShowingRoom = true
     @Published var isShowingStats = false
     @Published var isShowingError = false
     private(set) var error: Error?
-    private let accessTokenStore = TwilioAccessTokenStore()
     private var isAutoLayoutSwitchingEnabled = true
-    private var roomManager: RoomManager!
+    private var callManager: CallManager!
     private var speakerLayoutViewModel: SpeakerLayoutViewModel!
     private var subscriptions = Set<AnyCancellable>()
 
-    func configure(roomManager: RoomManager, speakerLayoutViewModel: SpeakerLayoutViewModel) {
-        self.roomManager = roomManager
+    func configure(callManager: CallManager, speakerLayoutViewModel: SpeakerLayoutViewModel) {
+        self.callManager = callManager
         self.speakerLayoutViewModel = speakerLayoutViewModel
 
-        roomManager.roomConnectPublisher
+        callManager.connectPublisher
```

Comment:
This view model now uses the CallKit layer instead of the TwilioVideo layer. I think it actually makes this a more proper view model. It is a little more focused on transforming data for UI display and has less non-UI logic.